va/validation-authority_test.go

```diff
@@ -332,6 +311,54 @@ func TestSimpleHttp(t *testing.T) {
 	test.AssertEquals(t, invalidChall.Error.Type, core.ConnectionProblem)
 }
 
+func TestSimpleHttpRedirectLookup(t *testing.T) {
```

Comment:
So we don't have a CNAME/DNAME redirect test as  `miekg/dns` doesn't follow aliases by itself, and we leave final resolution to `http.NewRequest` or `tls.DialWithDialer`. That seems appropriate to me.